package.sh

```diff
@@ -361,11 +446,13 @@ if [ -z "$NIGHTLY_BUILD" ]; then
             echo "Failed to create tag v$VERSION -- aborting"
             cleanup_exit 1
         fi
+        echo "Tag v$VERSION created ..."
         git push origin v$VERSION
         if [ $? -ne 0 ]; then
             echo "Failed to push tag v$VERSION to repo -- aborting"
             cleanup_exit 1
         fi
+        echo "and pushed to repo"
```

Comment:
Minor -- can you be more explicit here?

`echo "Tag v$VERSION pushed to report"`